### ISSUE TITLE
Provide lockscreen binding in default config

### ISSUE THREAD

**Issue by frsfnrrg** [swaywm/sway#2138]
Why? Most people setting up a screen locking shortcut will only configure it in the default/root mode,
so that the shortcut does not work inside a "resize" or other mode. This is insecure in the same way that X11's failure to lock the screen when a popup window (context menu, etc.) is active is.

The actual lockscreen program could be set in the "Variables" section of the default config file, and then triggered both in the  "resize" mode and in the default mode.

I'm not certain on what makes a good default (or even about this whole idea):
* $mod+Shift+l fits with the exit/reload scheme, but overlaps with the usual arrow keys (QASDWHJKL).
* $mod+Shift+Delete/Backspace when mistyped can ruin work and are inconsistently available
* $mod+Shift+Enter is reasonably safe (but might be triggered too easily by accident)
* $mod+Shift+Escape has no flaws I'm aware of

**Comment by RyanDwyer** [comment]
I don't think we should do this. Consider a new user who installs sway, is happily using it, then accidentally presses the key binding to lock their screen. Assuming we don't set a lockscreen background image, their screen would turn a solid colour and they would have no idea what's going on, and would probably reboot and lose their work. For this reason I think the lockscreen should not be configured by default, but a commented example in the config is fine.

**Comment by smlx** [comment]
I agree that a commented example configuration will probably suffice.

FWIW, I use $mod+Shift+`

**Comment by SirCmpwn** [comment]
I use

```
bindsym XF86Sleep exec $swaylock -i `wallpaper` --scaling fill
bindsym $mod+Pause exec $swaylock -i `wallpaper` --scaling fill
```
